### ISSUE TITLE
Chrome/FF developer tools element navigation is slow

### ISSUE THREAD

**Issue by ralphsmith80** [developit/preact-compat-example#11]
I've never seen this before so I'm not sure where the core issue is, but it seem to fit with this example as I created a preact app using `preact-cli` and `react-toolbox` so maybe you have some insight.

You can see the issue for yourself by navigation to [my app](https://fiday-4249d.firebaseapp.com/). You don't have to login, just open developer tools, select the Element panel, and start expanding elements. You should notice a long lag with the responsiveness of the element panel. 

I don't see this problem with the `react-toolbox` components [page](http://react-toolbox.com/#/components) or the `preact-compact-example` [page](https://preact-compat-example.surge.sh/). 

To attempt to isolate the issue I created a new app from preact-cli and only added the [`Snackbar`](http://react-toolbox.com/#/components/snackbar) example.  

What I noticed was that importing components as shown seemed to be the problem.
In other words changing this:

```
import { Button, Snackbar } from 'react-toolbox'
```

to this 

```
import { Button } from 'react-toolbox/lib/button'
import { Snackbar } from 'react-toolbox/lib/snackbar'
```

seemed to fix the problem. 

However after following that pattern for every `react-toolbox` component I'm still seeing the issue. I did introduce changes at top level routes which obviously worked, but seemed to have diminishing returns and once I added the main app route back in it's back to being painfully slow.
